script.js

```diff
@@ -3,35 +3,67 @@ const idApp = {};
 
 // Event Listeners Function  
 idApp.eventListeners = function() {
-   // Event Listener #1 - for when the user first submits their preferred region/gender
+   //Cache $('main') jQuery selector as we will repeatedly use it
+   $main = $('main');
+
+   // Event Listener #1 (Window A) - for when the user first submits their preferred region/gender
    $('form').on('submit', function(event){
       event.preventDefault();
-      // Store the user's chosen gender and region in variables
-      const userGender = $('#gender option:checked').val()
-      const userRegion = $('#regions option:checked').val()
-      // Call the API function, passing in the user's choices as arguments 
-      idApp.apiCall(userGender, userRegion);
+      // Cache the user's chosen gender and region (can be re-used again in second API call)
+      $userGender = $('#gender option:checked').val()
+      $userRegion = $('#regions option:checked').val()
+      // Error Handling - for if user does not choose any options 
+      if (!$userGender || !$userRegion) {
```

Comment:
this is a great error message function! good use of the not (!) operator also, I didn't think of that!